.azure-pipelines/all.yml

```diff
@@ -9,14 +9,14 @@ pr: none
 trigger:
   branches:
     include:
-      - master
+    - master
 resources:
   repositories:
-    - repository: integrations-core
-      type: github
-      name: Datadog/integrations-core
-      ref: master
-      endpoint: DataDog
```

Comment:
There are some other tabulation changes to revert

Reply:
fixed